tuist/tuist
# Remove Fourier from the repository

### Short description 📝
This PR moves the final tasks to bash scripts and removes Fourier from the repository. Note that we can remove the dependency on Ruby yet since acceptance tests are still implemented in Ruby using the Cucumber framework. Once we migrate them to Swift, we'll be able to drop Ruby.

### How to test the changes locally 🧐
The following scripts should succeed:

- `make/tasks/tuist/build.sh`
- `make/tasks/tuist/test.sh`
- `make/tasks/workspace/acceptance-test.sh`
- `make/tasks/workspace/generate-fixture.sh`
- `make/tasks/workspace/benchmark.sh`

### Contributor checklist ✅


### Reviewer checklist ✅
